Vagrantfile

```diff
@@ -0,0 +1,54 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  
+config.vm.box = "hashicorp/bionic64"
+config.vm.network "forwarded_port", guest: 5000, host: 5000
+config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+    sudo apt-get update; sudo apt-get install -y build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+# TODO: Install pyenv prerequisites
+# TODO: Install pyenv
+    echo " installing pyenv."
+      if [ ! -d \"$HOME/.pyenv\" ]; then
+        git clone https://github.com/pyenv/pyenv.git $HOME/.pyenv                  
+      fi
+      echo "export pyenv root"  
+      # export PYENV_ROOT=\"$HOME//.pyenv/\"
+      # export PATH=\"$PYENV_ROOT/bin:$PATH\"
+        echo "Echoing step for error"
+      if [ -d \"$HOME/.pyenv\" ]; then
+          echo "pyenv already installed on filesystem."
+      fi
+      echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+      echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+      echo 'eval "$(pyenv init -)"' >> ~/.profile
+      source ~/.profile
+         
+           echo "End of pyenv"  
+           pyenv install 3.9.0
+           pyenv global 3.9.0 
+           echo "Adding Poetry"
+           cd /vagrant
+           curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+           source $HOME/.poetry/env
+           echo "Install Poetry"
+           sudo poetry env use ~/.pyenv/versions/3.9.0/bin/python3.9
+           sudo poetry install
+    
+SHELL
+config.trigger.after :up do |trigger|
+  trigger.name = "Launching App"
+  trigger.info = "Running the TODO app setup script"
+  trigger.run_remote = {privileged:false, inline: "
+  # Install dependencies and launch
+  # <your script here >
```

Comment:
Given these comments, it would be nice to have the `poetry install` step occur inside this trigger, after you've changed to the right directory. As it currently stands, any new dependencies added to your pyproject.toml won't be picked up unless the user runs a full provision again